src/rules/__tests__/a11y-remove-disable-tooltip.test.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const {RuleTester} = require('eslint')
+const rule = require('../a11y-remove-disable-tooltip')
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+})
+
+ruleTester.run('a11y-remove-disable-tooltip', rule, {
+  valid: [
+    `import {IconButton} from '@primer/react';
+    <IconButton icon={SearchIcon} aria-label="Search" />`,
+  ],
+  invalid: [
+    {
+      code: `<IconButton icon={SearchIcon} aria-label="Search" unsafeDisableTooltip />`,
+      output: `<IconButton icon={SearchIcon} aria-label="Search" />`,
+      errors: [
+        {
+          messageId: 'removeDisableTooltipProp',
+        },
+      ],
+    },
+    {
+      code: `<IconButton icon={SearchIcon} aria-label="Search" unsafeDisableTooltip={true} />`,
+      output: `<IconButton icon={SearchIcon} aria-label="Search" />`,
+      errors: [
+        {
+          messageId: 'removeDisableTooltipProp',
+        },
+      ],
+    },
+  ],
+})
```

Comment:
(After we change the default value of unsafeDisableTooltip), Do you have a preference for `unsafeDisableTooltip={false}` vs not having it all? Should we add that to the linter?

Reply:
To add to this, is the plan to deprecate the prop in the future once we know `unsafeDisableTooltip={true}` isn't being used anywhere in dotcom (or at least a very low amount of usage)?

Reply:
>Do you have a preference for unsafeDisableTooltip={false} vs not having it all? Should we add that to the linter?

I really like the idea of adding it to the linter! I always imagined that we would remove the prop instead of `unsafeDisableTooltip={false}` - I'll add it to the rule.

>is the plan to deprecate the prop in the future once we know unsafeDisableTooltip={true} isn't being used anywhere in dotcom (or at least a very low amount of usage)?

This is the hope and the plan 🙌🏻 - however there might be some rare cases that we need to hide the tooltip and I guess we are yet to discover these cases as product teams test and remove `unsafeDisableTooltip={true}`. If we end up having icon buttons that we certainly don't need tooltips for whatever reason, we might choose to keep the prop.

Reply:
>I'll add it to the rule.

Actually, I just realised that the rule checks `unsafeDisableTooltip={false}` as well. Or in better words, it doesn't check the value of the prop 😆 I think this is fine because type safety will handle the value and we just need to worry about if the prop exists or not. I added a test case and into the docs though. Let me know if you have any concern 🙌🏻